### ISSUE TITLE
Create placeholder for lpc11uxx?

### ISSUE THREAD

**Issue by rnestler** [lpc-rs/lpc-placeholders#3]
Similar to #2 
@hannobraun Should we also deprecate https://crates.io/crates/lpc11uxx and create a lpc11uxx-pac crate instead? cc @dbrgn

**Comment by hannobraun** [comment]
Yes, I believe we should, for the reasons outlined in the README.
